curvlinops/inverse.py

```diff
@@ -197,3 +204,102 @@ def _matvec(self, x: ndarray) -> ndarray:
                 )
 
         return self._scale * result
+
+
+class KFACInverseLinearOperator(_InverseLinearOperator):
+    """Class to invert instances of the ``KFACLinearOperator``."""
+
+    def __init__(self, A: KFACLinearOperator, cache: bool = True):
```

Comment:
Yea, I also considered some functionality to add damping. I haven't implemented anything yet because I was not sure if we should only support adding a fixed value, or use a form of adaptive damping (as used by Kazuki in `ASDL`). For now we can follow your suggestion though.